cluster.go

```diff
@@ -81,11 +100,17 @@ func (c *cluster) newNode() *node {
 		env[m[1]] = m[2]
 	}
 
-	node := newNode(name, args, env, true,
+	node := newNode(
+		name,
+		args,
+		env,
+		true,
```

Comment:
That's a bit excessive. We didn't mandate this formatting in the cockroach code because not everyone agrees with it, including me. I'd reserve it for new code you right. Changing this in existing code is a bit aggressive.